### PR TITLE
Allow to increase timeout and to specify wait strategy

### PR DESCRIPTION
**Pull request by alexon1234**
## Description

Allow to customize the timeout for the plugin and the wait strategy for appending data into Tinybird

### CODE REVIEW

File: tests/test_tinybird.py [hunk 1]
```diff
@@ -17,15 +18,18 @@ def test_flaky():
     tinybird_url = os.environ["TINYBIRD_URL"] = 'https://fake-api.tinybird.co'
     datasource_name = os.environ["TINYBIRD_DATASOURCE"] = "test_datasource"
     tinybird_token = os.environ["TINYBIRD_TOKEN"] = 'test_token'
+    timeout = os.environ["TINYBIRD_TIMEOUT"] = "10"
+    wait = os.environ["TINYBIRD_WAIT"] = random.choice(['true', 'false'])
 
     with mock.patch('requests.post') as mock_post:
-        mock_post.return_value.status_code = 202
+        mock_post.return_value.status_code = 200 if wait == 'true' else 200
```
Comment (alrocar) [inline tests/test_tinybird.py]: else 202?

**Comment by alrocar** [comment]
I'm closing this one xD then https://github.com/tinybirdco/pytest-tinybird/pull/9

**Review by alrocar — APPROVED** [review]
minor comment but LGTM. Thanks!

File: tests/test_tinybird.py (hunk 1 above)
Comment (alexon1234) [reply tests/test_tinybird.py]: True

**Pull request merged**